src/ui/sass/modules/_SearchBar.scss

```diff
@@ -233,11 +233,18 @@ $searchbar-button-text-color-active: var(--yxt-searchbar-button-text-color-base)
     }
   }
 
+  &-voiceIconWrapper
+  {
+    width: 32px;
```

Comment:
Icon scss file have some width and height styling (sm and lg). Can we not use that here or is it necessary to have a specific width here?

Reply:
oh, you're right. i just tried that and that works better. Updating!